tests/lib/retry.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+snap_install() {
+    local PARAMS=$@
+    local CONDITIONS='timeout \| connection refused'
+    local RETRIES=2
+    local SLEEP=2
+
+    local ITER=0
+    until [ $ITER -gt $RETRIES ]; do
+        ERROR=$(snap install $PARAMS 2>&1> /dev/null)
```

Comment:
Maybe you meant `2>&1 >` ? shellcheck complains here for me: `Syntax error: redirection unexpected`